### ISSUE TITLE
Context-sensitive navigation to webpage regions of given types

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#195]
**Reported by benjaminhawkeslewis on 2008-10-11 12:31**
Allow rapid navigation to regions of a given type within webpages, web applications and widgets, using lists and/or next/last shortcut keys.

Key types include: navigation areas, main content, articles, and search functionality. These would be contextualized if you were within a particular widget.

Potential markup to support this includes HTML4 markup (MENU, DIR, REL="search" pointing to an internal fragment), proposed ARIA landmark roles ("navigation", "main", "search"), proprietary extensions (INPUT TYPE="search") and proposed HTML5 markup (NAV, SECTION, ARTICLE).

Labels could be derived for regions from headings, TITLE attributes, and proposed ARIA markup like LABELLEDBY.

Heuristics might be able to provide similar functionality even in pages without this markup.

Compare discussion of how ARIA landmark roles should be used and interpreted at:
- http://preview.tinyurl.com/3zlyfb [dev-accessibility](Mozilla)
- http://preview.tinyurl.com/4lcxr5 [wai-xtech]

**Blocked by #192**

**Comment by derekriemer** [comment]
Is this still relevant?

**Comment by ehollig** [comment]
Bringing this up again. Is this still relevant, or is this resolved from #192? This could also not be entirely clear on what this ticket is about,.

**Comment by Adriani90** [comment]
Articles are now navigable, you can assign a gesture for it and enable this in document formatting settings. Main navigation areas can be navigated by d for landmarks or m for regions. Search field can easilib be reached by pressing e for edit field. Thus, I am closing this issue since NVDA already provides this feature. I there is something still missing here, please create a new issue for the specific navigation area that you think would be useful to have contained into the quick navigation.
